systers_portal/.env.example

```diff
@@ -0,0 +1,20 @@
+SECRET_KEY = foobarbaz
+
+GOOGLE_MAPS_API_KEY= wqibceubvuywvibiwe3bb
+
+DEBUG = True
+
```

Comment:
nit: no need to add spaces for this, just bunch the above four all of them up together.